### ISSUE TITLE
Auto-refresh not working

### ISSUE THREAD

**Issue by talentedmrripley** [AHAAAAAAA/PokemonGo-Map#593]
The map doesn't update without a manual reload.  Tested with Chrome on Windows and Safari on iPhone.

**Comment by Hardstylah** [comment]
are you sure you added -ar 15 ? Because mine is working fine.

**Comment by mkody** [comment]
At least on develop, I can confirm I don't see requests coming from the browser with `-ar 10`.

**Comment by Hardstylah** [comment]
develop? I think im missing something. Diffrent versions? I dont even know which I have.

**Comment by ireun** [comment]
Same here, i don't see any refreshing in Chrome with `-ar 10`

**Comment by mkody** [comment]
Yup, do a `git checkout develop`. Master is not supported for now.

**Comment by craigbaines** [comment]
@mkody Could you help me out on using the develop branch, i have the master all up an running using the example.py. I attempted the develop one before but was lost lol.

**Comment by dorian-marchal** [comment]
Same here, on `develop` branch : `-ar 2` does not work.

**Comment by tylerstokes** [comment]
This works for me. You wont notice the normal browser refresh but you will notice pokemon appearing and disappearing

**Comment by tstokes8040** [comment]
This works for me. You wont notice the normal browser refresh but you will notice pokemon appearing and disappearing. I am using -ar 5

**Comment by mkody** [comment]
Quickly looking through the code, there isn't any `args.auto-refresh` referenced in the `develop` branch. So the -ar argument isn't actually used. :T

**Comment by theblackmarmoset** [comment]
@talentedmrripley Hey, you said you tested it on safari with your phone.  How do you use this on safari with the iphone?  I can't seem to get mine working.  Sorry I'm pretty much a mega noob when it comes to this type of stuff.

**Comment by talentedmrripley** [comment]
@theblackmarmoset I'm going to assume that you want to be able to access it when you're not at home.  To do that you need to launch the app with the -H 0.0.0.0

I'm also assuming that you're behind a router, so you'll need to do a couple things.
1.) Go to www.ipchicken.com and find out what your public IP address is.
2.) Forward a port from your router to the computer running the app.  Go to http://portforward.com/ and find your router model and follow the directions to forward port 5000.
3.) When you're out and about open Safari, or any other browser on your phone, and go to http://yourpublicip:5000
4.) Be a god amongst your friends and catch many new Pokemon.

**Comment by jabelone** [comment]
Auto refresh is working completely fine for me.  Is this still a bug?

**Comment by MikeDX** [comment]
I think this should be closed / marked as invalid as the most recent develop branch has auto page refresh every 10 seconds via ajax (however it does flash the pokemon briefly which is rather annoying)

**Comment by jabelone** [comment]
I think the poke flash is good, it signifies when it updates.  (but each to their own)
